src/model/event.rs

```diff
@@ -1979,6 +2025,101 @@ pub enum EventType {
     Other(String),
 }
 
+impl From<&Event> for EventType {
+    fn from(event: &Event) -> EventType {
+        event.event_type()
+    }
+}
+
+impl EventType {
+    const CHANNEL_CREATE: &'static str = "CHANNEL_CREATE";
```

Comment:
The compiler can deduce the lifetime of references in `const`s and `static`s by itself. You should remove the `'static` lifetimes to make the code more aesthetically appealing.
```suggestion
    const CHANNEL_CREATE: &str = "CHANNEL_CREATE";
```